### PR TITLE
Fix: Control planes can be deleted.

### PR DESCRIPTION
**Pull request by Guaris**
https://konghq.atlassian.net/browse/DOCU-4017

### CODE REVIEW

**Review by lena-larionova — CHANGES_REQUESTED** [review]
Can you please add a changelog entry as well?

File: app/konnect/gateway-manager/index.md [hunk 1]
```diff
@@ -56,9 +56,6 @@ entity-specific permissions. For more information, see [Administer teams](/konne
 
 ### {{site.base_gateway}} control planes
 
-Every region in every organization starts with one default control plane.
```
Comment (lena-larionova) [inline app/konnect/gateway-manager/index.md:59]: Why delete the first line? It's still accurate.

**Review by lena-larionova — COMMENTED** [review]
Please also remove the line here: https://docs.konghq.com/konnect/gateway-manager/#deleting-a-control-plane

"You cannot delete the default control plane."

File: app/konnect/gateway-manager/index.md (hunk 1 above)
Comment (mheap) [reply app/konnect/gateway-manager/index.md:59]: I think I saw that we're going to stop provisioning the default CP too

Comment (lena-larionova) [reply app/konnect/gateway-manager/index.md:59]: Looks like that hasn't gone live yet: 

> we will also stop provisioning the default control plane in each region soon 

https://kongstrong.slack.com/archives/C04RXLGNB6K/p1723651386893509?thread_ts=1723650140.813759&cid=C04RXLGNB6K

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**